docs/advanced-usage.md

```diff
@@ -104,6 +104,42 @@ jobs:
       - run: npm test
 ```
 
+## Nightly versions
+
+You can specify a nightly version to download it from https://nodejs.org/download/nightly.
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Node sample
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.0.0-nightly' # or 16-nightly
```

Comment:
To avoid ambiguity, I think it's worth a separate explanation of what the `16-nightly` syntax means.  From my understanding, it means the same thing as saying `node-version: 16` but for nightlies.  So, it will find the nightly for the latest Node 16 (as opposed to 16.0.0).